### PR TITLE
Add GitHub Action to run tests

### PR DESCRIPTION
**Pull request by anaulin**
An attempt at a simple CI setup for this repo. Meant to run tests on PRs and main-merges.

We can add other stuff (e.g. linting) if this works well.

### CODE REVIEW

**Comment by anaulin** [comment]
Example successful workflow run: https://github.com/EscapeRoomsCommunity/escape_roomba/pull/8/checks?check_run_id=1178678960 ✅

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
```
Comment (egnor) [inline .github/workflows/python-app.yml]: (the blank lines around "runs-on:" look slightly odd to my eye, but totally nbd)

**Review by egnor — APPROVED** [review]
Super sexy!

File: .github/workflows/python-app.yml (hunk 1 above)
Comment (anaulin) [reply .github/workflows/python-app.yml]: Fixed.

**Pull request merged**